lib/aca_entities/pdm/pdm.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module AcaEntities
+  # Domain model entities, contracts and associated objects related to
+  # PDM (periodic data matching) proceses like PVC and RRV
+  #
+  module Pdm
+    # shared dependencies
+    # types
+    require 'aca_entities/types'
+
+    # external dependencies
+    require 'aca_entities/timestamp'
+
+    require_relative 'types'
```

Comment:
Is this different from the line above that has `require 'aca_entities/types'`?  Can we remove if it's redundant?